external/fv3fit/fv3fit/keras/_models/convolutional.py

```diff
@@ -125,19 +111,8 @@ def train_convolutional_model(
     return predictor
 
 
-def _count_array_features(arr: np.ndarray) -> int:
-    """
-    Given a 3/4-d [sample, x, y(, z)] array, return its number of
-    vertical levels.
-    """
-    if len(arr.shape) == 3:
-        return 1
-    else:
-        return arr.shape[3]
-
-
 def _ensure_4d(array: np.ndarray) -> np.ndarray:
-    # ensures [sample, x, y, z] dimensionality
+    # Ensures [sample, x, y, z] dimensionality.
```

Comment:
You don't need to change it back, but I prefer the other style for short declarations. Even with punctuation, this isn't a complete sentence because it has no subject.